examples/fastapi/functional_test/app.py

```diff
@@ -0,0 +1,27 @@
+import authomatic
+from fastapi import FastAPI
+from fastapi.responses import PlainTextResponse
+from authomatic.adapters import DjangoAdapter

```

Comment:
Shouldn't this be FastAPIAdapter? (see also line 21)

Reply:
Yes, mb, it still passes the tests, I solved it and I'm gonna commit after you review liveandletdie PR because the next commit here depends on it